src/react-queries/queries/common.ts

```diff
@@ -78,15 +78,18 @@ export const common = createQueryKeys('common', {
       | typeof getThorchainLpPosition
   }) => ({
     queryKey: ['thorchainFromAddress', accountId, assetId, opportunityId],
-    queryFn: async () =>
-      await getThorchainFromAddress({
+    queryFn: async () => {
+      if (!accountId) throw new Error('accountId is required')
+
+      return await getThorchainFromAddress({
```

Comment:
And revert this guy too, this is the responsibility of consumers to pass a valid payload